packages/stratimikos/adapters/belos/src/BelosThyraAdapter.hpp

```diff
@@ -275,8 +278,9 @@ namespace Belos {
          const ScalarType beta, TMVB& mv )
     {
       using Teuchos::arrayView; using Teuchos::arcpFromArrayView;
-
+#ifdef HAVE_STRATIMIKOS_BELOS_TIMERS
       Teuchos::TimeMonitor tM(*Teuchos::TimeMonitor::getNewTimer(std::string("Belos::MVT::MvTimesMatAddMv")));
+#endif
```

Comment:
I am wondering if it might be more maintainable to define a macro for Stratimikos called something like `STRATIMIKOS_TIME_MONITOR()` which would replace this block of code with:

```
STRATIMIKOS_TIME_MONITOR("Belos::MVT::MvTimesMatAddMv");
```

In fact, I am kind of surprised that someone wrote lots of messy code like:

```
Teuchos::TimeMonitor tM(*Teuchos::TimeMonitor::getNewTimer(std::string("Belos::MVT::MvTimesMatAddMv")));
```

all over the place.

The definition of the macro would be:

```
#ifdef HAVE_STRATIMIKOS_BELOS_TIMERS
#  define STRATIMIKOS_TIME_MONITOR(NAME) \
    Teuchos::TimeMonitor tM(*Teuchos::TimeMonitor::getNewTimer(std::string(NAME)))
#else
#  define STRATIMIKOS_TIME_MONITOR(NAME)
#endif
```